packages/runtime/datastore/src/channelContext.ts

```diff
@@ -85,7 +83,7 @@ export function createChannelServiceEndpoints(
 	submitFn: (content: unknown, localOpMetadata: unknown) => void,
 	dirtyFn: () => void,
 	isAttachedAndVisible: () => boolean,
-	storageService: IDocumentStorageService,
+	storageService: IRuntimeStorageService,
```

Comment:
i don't think any layers below the container runtime itself need anything but `readblob`, since we're changing it, i think it would be worth it to use a smaller type from container runtime down.

Reply:
probably have to do the same thing you do with IRuntimeStorageService, where you make a whole copy, and then deprecate everything, but readblob

Reply:
Good point. Let me take a look at usages at various layers. Maybe we can keep shriking it down. I may decide to do it in multiple PRs though just so it doesn't get too big.

Reply:
Updated the PR as per your suggestion.